packages/ui/src/components/TextField/TextField.tsx

```diff
@@ -170,6 +182,19 @@ export const TextField = (props: TextFieldProps) => {
   const [_value, setValue] = React.useState<Value>(value ?? '');
   const theme = useTheme();
 
+  const sxTooltipIconLeft = {
+    marginRight: `${theme.spacingFunction(4)}`,
+    padding: `${theme.spacingFunction(4)} ${theme.spacingFunction(4)} ${theme.spacingFunction(4)} ${theme.spacingFunction(2)}`,
+    '&& svg': {
+      color: `${theme.tokens.color.Neutrals[90]}`,
```

Comment:
I believe we have to use `theme.tokens.component.Label` for the Icon and texts?
```suggestion
      color: `${theme.tokens.components.Label}`,
```

Reply:
@harsh-akamai Thank you, I've updated this and further defined the styles used in the light and dark themes. Please review the `light.ts` and `dark.ts` files.